addons/mail/tests/test_mail_activity.py

```diff
@@ -1,10 +1,101 @@
 from freezegun import freeze_time
 
 from odoo import exceptions
+from odoo import fields
+from odoo.addons.mail.tests.common import MailCommon
 from odoo.addons.mail.tests.common_activity import ActivityScheduleCase
 from odoo.tests import tagged, HttpCase
 
 
+@tagged('post_install', '-at_install')
```

Comment:
Would keep the default here (i.e. no tags, i.e. at_install currently)

post_install is only necessary below because it's an HttpCase, and the test stack is such that these need to be run post_install.